frontend/packages/console-app/src/components/user-preferences/__tests__/UserPreferencesPage.spec.tsx

```diff
@@ -1,17 +1,33 @@
-import * as React from 'react';
-import { Tabs } from '@patternfly/react-core';
-import { shallow, ShallowWrapper } from 'enzyme';
+import { screen, act } from '@testing-library/react';
 import * as Router from 'react-router-dom-v5-compat';
 import { useResolvedExtensions } from '@console/dynamic-plugin-sdk';
-import { LoadingBox } from '@console/internal/components/utils';
 import { useExtensions } from '@console/plugin-sdk/src';
 import { useQueryParams } from '@console/shared/src';
+import { renderWithProviders } from '@console/shared/src/test-utils/unit-test-utils';
 import UserPreferencePage from '../UserPreferencePage';
 import {
   mockUserPreferenceGroupExtensions,
   mockUserPreferenceItemExtensions,
 } from './userPreferences.data';
 
+jest.mock('@console/shared/src/components/document-title/DocumentTitle', () => ({
+  DocumentTitle: ({ children }) => children,
```

Comment:
unless we are testing some specific title the documenttitle mock could just be jest.fn because it sticks stuff into the `head` and not `body`

Reply:
@logonoff Good catch. But `jest.fn()` returns `undefined` by default, which will cause the tests to fail. However since `DocumentTitle` needs JSX, returning `null` should work.